### ISSUE TITLE
Issues with Images scaling on PDF documentation

### ISSUE THREAD

**Issue by mickaelandrieu** [puli/issues#106]
According to readthedocs docs (lol), the solution is to add PIL as a requirements:

> Image scaling in docutils depends on PIL. PIL is installed in the system that RTD runs on. However, if you are using the virtualenv building option, you will likely need to include PIL in your requirements for your project.

But: I don't know what is PIL, and no others docs on readthedocs talk about it.

What do you think ?

**Comment by webmozart** [comment]
Hi! Could it be you posted this on the wrong issue tracker? :)

**Comment by mickaelandrieu** [comment]
I don't think so: rendering of some images of the pdf generated of docs.puli.io have issues. 

Or maybe docs issues have to be reported in another place?

**Comment by webmozart** [comment]
Oh I see, you didn't mention docs.puli.io ;) Could you describe what exactly your problem is?

**Comment by mickaelandrieu** [comment]
@webmozart yes, for instance the first page with the puli logo is totaly broken: the logo seems to be outside the page (in pdf rendering).

The first schema (Resources discovery) is very big in pdf rendering compared to the web rendering of this data.

Sounds like a problem with images scaling, correct ?

I have update the issue label ;)

**Comment by webmozart** [comment]
Ah, you're talking about the PDF. Sorry, I didn't get that. :)

**Comment by holloway87** [comment]
It could be that you're looking for the Python Image Library: http://www.pythonware.com/products/pil/
